test/other/test_override_stub.out

```diff
@@ -0,0 +1 @@
+1
```

Comment:
How about `printf("override called");` in the fork override.. then the `.out` file will be more meaningful maybe?

Reply:
Doesn't "1" get print *after* override is called?